.github/ISSUE_TEMPLATE/docs-request.yaml

```
@@ -0,0 +1,45 @@
+name: "[Internal] Dcumentation request"
+description: Request a documentation change or enhancement for any supported release (Elastic employees).
+title: "[REQUEST]: "
+labels: ["request", "Team:Docs"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this request! This form will create an issue that the Platform Docs team will triage and prioritize.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe what needs to be documented.
+      placeholder: I need...
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Resources
+      description: Please link to as many related resources as possible. Think about issues, PRs, demos, Slack threads, existing or related docs pages, etc.
+      placeholder: How can the docs team learn more about this feature?
+    validations:
+      required: true
+  - type: dropdown
+    id: collaboration
+    attributes:
+      label: Collaboration
+      description: Choose the expected collaboration model
+      options:
+        - "The documentation team will investigate the issue and create the initial content."
```

```suggestion
        - "Choose a collaboration model."
        - "The documentation team will investigate the issue and create the initial content."
```
I think we should avoid `The documentation team will investigate the issue and create the initial content` as the default. If people opening issues want writers to be responsible for the investigation and initial draft, it seems best to me that they actively, explicitly select that option.